maintenance/projects/js-toolkit/packages/liferay-npm-build-tools-common/src/project/index.ts

```diff
@@ -516,7 +516,7 @@ export class Project {
 			const linkTarget = fs.readlinkSync(this._toolsDir.asNative);
 
 			this._toolsDir = new FilePath(
-				linkTarget.startsWith('/')
+				process.platform === 'win32' && /^[A-Z]:\\/.test(linkTarget) || linkTarget.startsWith('/')
```

Comment:
How about using https://nodejs.org/api/path.html#pathisabsolutepath instead of using a regexp?

Reply:
Sounds great. And obviously it works. Updated the PR.

Reply:
Perfect then :+1. Thx :-)